docs/snippets/react/portable-stories-jest-compose-story.ts.mdx

```diff
@@ -0,0 +1,19 @@
+```tsx
+// Button.test.tsx
+import { jest, test, expect } from 'jest';
+import { render, screen } from '@testing-library/react';
+import { composeStory } from '@storybook/react';
+
+import meta, { Primary } from './Button.stories';
+
+test('onclick handler is called', () => {
+  // Returns a story which already contains all annotations from story, meta and global levels
+  const PrimaryStory = composeStory(Primary, meta);
+
+  const onClickSpy = jest.fn();
```

Comment:
@yannbf — Did we want to use mocked args here (and all the other similar snippets) instead?

Reply:
I guess not, it will probably make things more confusing